src/lib.rs

```diff
@@ -219,4 +228,15 @@ impl Builder {
 
     self
   }
+
+  pub fn set_exlude_windows(mut self, exlude_windows: &[&str]) -> Self {
+    if !exlude_windows.is_empty() {
+      let mut exlude_set: HashSet<String> = HashSet::with_capacity(exlude_windows.len());
+      for win in exlude_windows {
+        exlude_set.insert(win.to_string());
+      }
+      self.exlude_windows = Some(exlude_set);
+    }
+    self
+  }
```

Comment:
could you move this and `set_auto_show` to be above the `build` method?

Reply:
and also rename them to `with_*` instead of `set_*`

Reply:
@amrbashir  i have fix that